apps/filecoin-site/src/app/_components/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+  DescriptionText,
+  type DescriptionProps,
+} from '@filecoin-foundation/ui/DescriptionText'
+
+import { Button } from '@/components/Button'
+import { Kicker, type KickerProps } from '@/components/Kicker'
+
+import { Title, type TitleProps } from './Title'
+
+type PageHeaderProps = {
+  title: TitleProps['children']
+  kicker?: KickerProps['children']
+  description?: DescriptionProps['children']
+  cta?: React.ReactElement<typeof Button>
+}
+
+export function PageHeader({
+  title,
+  kicker,
+  description,
+  cta,
+}: PageHeaderProps) {
+  return (
+    <header>
```

Comment:
Chances are this element would take full width on mobile.

```suggestion
    <header className="flex flex-col gap-10">
```